practice-two/src/index.html

```diff
@@ -51,7 +51,37 @@ <h1 class="navbar__logo">H.</h1>
         </nav>
 
         <!-- START BANNER -->
-
+        <section class="banner-section">
+          <hgroup class="banner-section__header">
+            <div class="banner-section__header-intro">
+              <p class="banner-section__header-text">RESPONSIVE UI KIT</p>
+            </div>
+            <h1 class="banner-section__header-title">HOFMANN</h1>
+          </hgroup>
+          <article class="banner-section__description">
+            <p class="banner-section__description-title">We’re Bring It To The New Level</p>
+            <p class="banner-section__description-text text-normal">Every new computer that’s brought home from the store has an operating system installed onto it. But what most new computer users don’t realize, is that without an operating system, that computer would be a simple shell of possibilities.</p>
+          </article>
+          <figure class="banner-section__image">
+            <div>
+              <img src="assets/images/iphone.png" alt="iphone image" class="banner-section__image--iphone">
+            </div>
+            <div>
+              <img src="assets/images/iphone-screen.png" alt="iphone screen image" class="banner-section__image--iphone-screen">
+            </div>
+            <div class="banner-section__image--iphone-app">
+              <img src="assets/images/iphone-app.png" alt="iphone app image" class="img-default">
+            </div>
+          </figure>
+          <div class="banner-section__apps">
+            <div class="banner-section__app-dowload">
```

Comment:
No need to use a div element here. You can remove it, and set display inline for `<a>` tag